v2/thirdpartyemailpassword/common-customizations/embed-sign-in-up-form.mdx

```diff
@@ -210,53 +493,214 @@ function MyLandingPage() {
 </TabItem>
 </FrontendPreBuiltUITabs>
 
-## Step 3: Changing the website path for the login UI
+:::note
+In the above case, redirection may occur if a claim fails. For instance, in the case of an Email Verification claim, if the user's email is not verified, they will be redirected to the email verification page. To prevent redirection for failed claims, please contact us on [Discord](https://supertokens.com/discord) for assistance.
+:::
 
-The default path for this is component is `/{websiteBasePath}/`.
+### Rendering the Auth Widget in a popup
 
-If you are displaying this at some custom path, then you need add additional config on frontend:
+The following example shows the scenario where you embed the Auth Widget in a popup, and upon successful login, you aim to close the popup.
 
 <FrontendPreBuiltUITabs>
 <TabItem value="reactjs">
+<Question
+    question="Do you use react-router-dom?">
+<Answer title="Yes">
+
+<RRDVersionSubTabs>
+
+<TabItem value="v6">
 
 ```tsx
+import React, { useState, useEffect } from "react";
+import Modal from "react-modal";
 import SuperTokens from "supertokens-auth-react";
 import ThirdPartyEmailPassword from "supertokens-auth-react/recipe/thirdpartyemailpassword";
+// highlight-start
+import { SignInAndUp } from "supertokens-auth-react/recipe/thirdpartyemailpassword/prebuiltui";
+import Session from "supertokens-auth-react/recipe/session";
+// highlight-end
+import { useNavigate } from "react-router-dom";
 
 SuperTokens.init({
     appInfo: {
         apiDomain: "...",
         appName: "...",
-        websiteDomain: "...",
+        websiteDomain: "..."
     },
     recipeList: [
         ThirdPartyEmailPassword.init({
             // ...
         }),
+        // ...
     ],
-    //highlight-start
-    // The user will be taken to the custom path when then need to login.
-    getRedirectionURL: async (context) => {
+    // highlight-start
+    async getRedirectionURL(context) {
+        // The user will be taken to this path when they need to login.
         if (context.action === "TO_AUTH") {
-            return "/custom-login-path";
+            return "/auth?showModal=true"; // return the path where you are rendering the Auth UI
         };
+    },
+    // highlight-end
+});
+
+function AuthPopup() {
+  let sessionContext = Session.useSessionContext();
+  const navigate = useNavigate();
+  const urlParams = new URLSearchParams(window.location.search);
+  const [isModalOpen, setIsModalOpen] = useState(urlParams.get("showModal") === "true");
```

Comment:
No need to add the urlParams thing. Keep it simple. Set it to false, and in the snippet, call openModal in case a session does not exist in a useEffect.

Reply:
done